src/sass/landing_pages/usb.scss

```diff
@@ -0,0 +1,423 @@
+}
+
+.separator {
+    margin-top: 10px;
+    margin-bottom: 10px;
+    height: 1px;
+}
+
+.separator-md {
+    height: 20px;
+}
+
+.separator-lg {
+    height: 30px;
+}
+
+.separator-xl {
+    height: 50px;
+}
+
+.bottom-white-border {
+    border-bottom: 1px solid #ffffff;
+}
+
+.right-white-border {
+    border-right: 1px solid #ffffff;
+}
+
+.icon-xl {
+    display: inline-block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 20px;
+    width: 150px;
+    max-width: 150%;
+    height: 150px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+}
+
+.icon-buy {
+    background: url('../images/usb_page/buy.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-trade {
+    background: url('../images/usb_page/trade.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-clients {
+    background: url('../images/usb_page/clients.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-contract {
+    background: url('../images/usb_page/contract.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-crypto-traders {
+    background: url('../images/usb_page/crypto-traders.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-hold {
+    background: url('../images/usb_page/hold.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-online-business {
+    background: url('../images/usb_page/online-business.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-peaceofmind {
+    background: url('../images/usb_page/peaceofmind.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-redeem {
+    background: url('../images/usb_page/redeem.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-redeem-usd {
+    background: url('../images/usb_page/redeem-usd.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-source-code {
+    background: url('../images/usb_page/source-code.svg') no-repeat;
+    background-position: center;
+}
+
+.icon-supply {
+    background: url('../images/usb_page/supply.svg') no-repeat;
+    background-position: center;
+}
+
+/* SVG */
+
+.window {
+    fill: #e98024;
+    stroke: #e98024;
+}
+
+/* Video Embed */
+
+.video-container {
+    width: 100%;
+    height: 0px;
+    position: relative;
+    padding-bottom: 56.19%;
+    //box-shadow: 0 3px 28px 0 rgba(0, 0, 0, 0.03), 0 3px 28px 0 rgba(0, 0, 0, 0.3);
+}
+
+.video-container iframe, .video-container object, .video-container embed {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+}
+
+/* Modifiers */
+
+.title-xl {
+    font-size: 36px;
+}
+
+.ft-100 {
+    font-weight: 100;
+}
+
+.ft-300 {
+    font-weight: 300;
+}
+
+.ft-400 {
+    font-weight: 400;
+}
+
+.table-col {
+    border-bottom: 3px solid #f2f2f2;
+}
+
+.bottom-30 {
+    margin-bottom: 30px;
+}
+
+.bottom-50 {
+    margin-bottom: 50px;
+}
+
+.dark-grey-bg {
+    background: #eaeaea;
+}
+
+.justify-content-center {
```

Comment:
i think there's existing class that is equivalent to this: `.gr-row-align-center`, you can use this instead